INSTALL.rst

```diff
@@ -327,12 +334,20 @@ before you get started.
   conda create --name skimage-dev
   # Activate it
   conda activate skimage-dev
-  # Install major development and runtime dependencies of scikit-image
-  conda install --file requirements/default.txt --file requirements/build.txt --file requirements/test.txt
-  # Install scikit-image from source
-  pip install -e . -vv  ## TODO: to be updated for meson (see meson.md)
+  # Install main development and runtime dependencies
+  conda install -c conda-forge --file requirements.txt
+  # Install build dependencies of scikit-image
+  conda install -c conda-forge --file requirements/build.txt
+  # Build scikit-image from source
+  ./dev.py build
   # Test your installation
-  pytest --pyargs skimage
+  ./dev.py test
+  # Build docs
+  ./dev.py docs
+  # Try the new version in IPython
+  ./dev.py ipython
+
+For more information about building and using the dev.py package, see ``meson.md``.
```

Comment:
Thanks, @stefanv! Did you mean replacing ``meson.md`` with a link to the file?

Reply:
Oh, I just meant that those were working notes, and not truly intended for mass consumption.

Reply:
Oh, okay! I'm happy to remove it if it's preferable... Note that I didn't add this line: I just moved it from `CONTRIBUTING.rst` line 90 to `INSTALL.rst` line 350.